src/modules/users/controller.js

```diff
@@ -179,3 +200,94 @@ export const userAvatar = async (req, res) => {
     return res.sendStatus(500);
   }
 };
+
+export const userEmailVerify = async (req, res) => {
+  const verifyToken = req.params.verify;
+  console.log(verifyToken);
+  if (verifyToken.length === 0)
+    return res.status(400).json({
+      status: "Bad request",
+      code: 400,
+    });
+  const user = await UserService.findEmailToken(verifyToken);
+  if (user.length === 0)
+    return res.status(404).json({
+      status: "User not found",
+      code: 404,
+    });
```

Comment:
tutaj tak samo. 

Dodatkowo: tak naprawde sprawdzasz, czy `user` albo `verifyToken` to nie sa empty Arrays. Mozesz to zrobic poprzez sprawdzanie falsey, czyli `if (!user) {}` / `if (!verifyToken) {}`. Krocej ,bardzo standardowo, i latwo sie to czyta, bo dla czlowieka to jest jak "jesli nie ma usera/tokenu".